src/components/Datepicker/DatepickerSingleInput.spec.tsx

```diff
@@ -64,12 +64,12 @@ describe('DatepickerSingleInput', () => {
         const formattedDate = format(intendedDate, 'dd/MM/yy');
 
         // Simulate an input change
-        await userEvent.type(input, formattedDate);
+        userEvent.type(input, formattedDate);
 
         expect(mockErrorTextHandler).toHaveBeenCalledTimes(1);
```

Comment:
Thanks for bringing this up. In this case, I just removed the await from the userEvent, because the function isn't returning a promise that can be "awaited". I'd love to follow up with a PR to improve the tests as well. It looks like this huge change is a good opportunity to evaluate the existing code base.

Reply:
I've created #173 and will mark this as resolved